### PR TITLE
chore(EMS-3969): align test descriptions

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
Some tests had the following descriptions:

```ts
describe('when xyz returns success as true', () => { ... });
describe('when xyz returns success as false', () => { ... });

it('should display ...')
```

Whereas the majority use the following descriptions:

```ts
describe('when xyz returns success=true', () => { ... });
describe('when xyz returns success=false', () => { ... });

it('should render ...')
```

## Resolution :heavy_check_mark:
- Align test descriptions.

## Miscellaneous :heavy_plus_sign:
- Remove some unnecessary comments.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/validation/loss-payee-details-validation.spec.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ context('Insurance - Policy - Loss Payee Details - Validation', () => {
     const FIELD_ID = LOCATION;
     const ERROR = ERRORS[FIELD_ID];
 
-    it('should display validation error when radio is not selected', () => {
+    it('should render validation error when radio is not selected', () => {
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/validation/loss-payee-details-validation.spec.js:70]: Updated all 3x instances of this to say "a validation error"

File: e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/validation/loss-payee-details-validation.spec.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ context('Insurance - Policy - Loss Payee Details - Validation', () => {
     const FIELD_ID = LOCATION;
     const ERROR = ERRORS[FIELD_ID];
 
-    it('should display validation error when radio is not selected', () => {
+    it('should a render validation error when radio is not selected', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/validation/loss-payee-details-validation.spec.js:70]: typo

File: e2e-tests/insurance/cypress/e2e/journeys/policy/name-on-policy/validation/name-on-policy-validation.spec.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ context('Insurance - Policy - Name on policy - Validation', () => {
       cy.navigateToUrl(url);
     });
 
-    it('should display validation error', () => {
+    it('should a render validation error', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/name-on-policy/validation/name-on-policy-validation.spec.js:59]: typo

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js [hunk 4]
```diff
@@ -105,7 +105,7 @@ context(
         field(DIFFERENT_TRADING_NAME).input().should('not.be.visible');
       });
 
-      it(`should display conditional ${DIFFERENT_TRADING_NAME} input when selecting the trading name "yes" radio`, () => {
+      it(`should render conditional ${DIFFERENT_TRADING_NAME} input when selecting the trading name "yes" radio`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-page.spec.js:108]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-financial-year-end.spec.js [hunk 5]
```diff
@@ -55,7 +55,7 @@ context(`Insurance - Your business - Turnover page - when ${fieldId} exists`, ()
     cy.deleteApplication(referenceNumber);
   });
 
-  it(`should display ${FINANCIAL_YEAR_END_DATE} section`, () => {
+  it(`should render ${FINANCIAL_YEAR_END_DATE} section`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-financial-year-end.spec.js:58]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-page.spec.js [hunk 6]
```diff
@@ -74,7 +74,7 @@ context(
         cy.checkText(headingCaption(), CONTENT_STRINGS.HEADING_CAPTION);
       });
 
-      it(`should display ${FINANCIAL_YEAR_END_DATE} section`, () => {
+      it(`should render ${FINANCIAL_YEAR_END_DATE} section`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-page.spec.js:77]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/your-buyer/buyer-financial-information/buyer-financial-information-page.spec.js [hunk 7]
```diff
@@ -82,7 +82,7 @@ context(
         cy.checkRadioInputNoAriaLabel(CONTENT_STRINGS.PAGE_TITLE);
       });
 
-      it('should display summary text with collapsed conditional `details` content', () => {
+      it('should render summary text with collapsed conditional `details` content', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/your-buyer/buyer-financial-information/buyer-financial-information-page.spec.js:85]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/declarations/anti-bribery/code-of-conduct/anti-bribery-code-of-conduct.spec.js [hunk 8]
```diff
@@ -90,7 +90,7 @@ context(
         codeOfConductPage.revealText().should('not.be.visible');
       });
 
-      it('should display conditional `we will email you` hint when selecting the "yes" radio', () => {
+      it('should render conditional `we will email you` hint when selecting the "yes" radio', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/declarations/anti-bribery/code-of-conduct/anti-bribery-code-of-conduct.spec.js:93]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/export-contract/agent-charges/agent-charges.spec.js [hunk 9]
```diff
@@ -163,7 +163,7 @@ context(
             cy.assertAgentChargesFieldValues({ percentageMethod: true });
           });
 
-          it(`should display conditional "${PERCENTAGE_CHARGE}" field`, () => {
+          it(`should render conditional "${PERCENTAGE_CHARGE}" field`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/export-contract/agent-charges/agent-charges.spec.js:166]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/export-contract/how-was-the-contract-awarded/how-was-the-contract-awarded.spec.js [hunk 10]
```diff
@@ -107,7 +107,7 @@ context(
         fieldSelector(OTHER_AWARD_METHOD).input().should('not.be.visible');
       });
 
-      it(`should display conditional "${OTHER_AWARD_METHOD}" section when selecting the "${OTHER.VALUE}" radio`, () => {
+      it(`should render conditional "${OTHER_AWARD_METHOD}" section when selecting the "${OTHER.VALUE}" radio`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/export-contract/how-was-the-contract-awarded/how-was-the-contract-awarded.spec.js:110]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/different-name-on-policy-page.spec.js [hunk 11]
```diff
@@ -72,7 +72,7 @@ context(
         cy.checkText(fieldSelector(POLICY_CONTACT_DETAIL).hint(), CONTENT_STRINGS.HINT);
       });
 
-      it(`should display ${FIRST_NAME} field and be prepopulated`, () => {
+      it(`should render ${FIRST_NAME} field and be prepopulated`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/different-name-on-policy-page.spec.js:75]: should this be render the?

File: e2e-tests/insurance/cypress/e2e/journeys/policy/name-on-policy/name-on-policy-page.spec.js [hunk 12]
```diff
@@ -81,7 +81,7 @@ context(
         field(POSITION).input().should('not.be.visible');
       });
 
-      it(`should display conditional "${POSITION}" section when selecting the "yes" radio`, () => {
+      it(`should render conditional "${POSITION}" section when selecting the "yes" radio`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/name-on-policy/name-on-policy-page.spec.js:84]: should this be render the?

**Comment by ttbarnes** [comment]
@Zainzzkk Updated various descriptions.

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**